js/common.d.ts

```diff
@@ -288,6 +288,12 @@ export type ValidationStatus = 'valid' | 'invalid' | 'pending';
  */
 export type ValueChangedCallMode = 'onMoving' | 'onMovingComplete';
 
+/**
+ * @public
+ * @namespace DevExpress.common
+ */
+ export type ValueChangeMode = 'onHandleMove' | 'onHandleRelease';
```

Comment:
We should bring this name into line with recently added `ValueChangedCallMode`
/cc @vladaskorohodova could you take a look?

Reply:
This types cannot be merged because they are used in different components. We will rename `ValueChangedCallMode` in RangeSelector later.

Discussed it in a voice.